mail_template_attachment_i18n/models/ir_attachment_language.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2018 Onestein
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+@api.model
+def _lang_get(self):
+    return self.env['res.lang'].get_installed()
+
+
+class IrAttachmentLanguage(models.Model):
+    _name = 'ir.attachment.language'
+
+    mail_template_id = fields.Many2one(
+        comodel_name='mail.template',
+        string='Template',
+        required=True,
+        ondelete='cascade',
+    )
+
+    lang = fields.Selection(
+        selection=_lang_get,
```

Comment:
Why do a function instead of something like this:
`selection=lambda self: self.env['res.lang'].get_installed(),`
Or maybe put the `_lang_get()` into the class to allow overwrite (you have to place it after the `_name` of the class)